community/connectToDB.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose')
+const dotenv = require('dotenv')
+const config = require('./src/Config/config')
+const path = require('path')
+
+dotenv.config({ path: path.resolve(__dirname, '.env') })
+
+const connectToDB = async () => {
+  const connectionParams = {
+    useNewUrlParser: true,
+    useUnifiedTopology: true
+  }
```

Comment:
i don't think you need to set these params since you are using version higher than 6 `"mongoose": "^8.1.3"`